docs/visual-testing/integrations/nightwatch.md

```diff
@@ -355,6 +355,13 @@ module.exports = {
 };
 ```
 
+### Skip Execution of Visual Tests
+
+In the event you're running / debugging tests locally and would like to disable the Sauce Visual integration temporarily, you can use the `SAUCE_VISUAL_SKIP` environment variable with any truthy value. When provided, we'll skip build creation, taking snapshots, and all snapshot based assertions will be marked as successful.
```

Comment:
I'm playing devil advocate role now - what is truth value? Maybe we can show an example?

```suggestion
In the event you're running / debugging tests locally and would like to disable the Sauce Visual integration temporarily, you can use the `SAUCE_VISUAL_SKIP` environment variable with any truthy value. When provided, we'll skip build creation, taking snapshots, and all snapshot based assertions will be marked as successful.

```bash
export SAUCE_VISUAL_SKIP=true
```
```

Reply:
I considered putting an example... But each example of actually setting the ENV would differ depending on the OS. Figured keeping it as simple as possible was the way to go. We also don't document each ENV in the table above specifically (though this isn't added there yet since it's not in every binding yet).